random.js

```diff
@@ -0,0 +1,76 @@
+var slideIndex = 0;
+var slides = document.getElementsByClassName("mySlides");
+var dots = document.getElementsByClassName("slide-inactive");
+var i;
+var userClick = false;
+let myTimeout;
+
+function currentSlide(n){
+    showSlides(slideIndex = n);
+}
+
+let showSlides = n => {
+    slideIndex > slides.length ? slideIndex = 1 : false;
+    n < 1 ? slideIndex = slides.length : false;
+
+    for(i = 0; i < slides.length; i++){
+        slides[i].style.display = "none";
+    }
+    for (i = 0; i < dots.length; i++) {
+        reAssign();
+    }
+    slides[slideIndex-1].style.display = "flex";
+    dots[slideIndex-1].className += " slide-active";
+}
+
+let slideAutomatic = () => {
+    for (i = 0; i < slides.length; i++) {
+        slides[i].style.display = "none";  
+    }   
+
+    for (i = 0; i < dots.length; i++) {
+        reAssign();
+    }
+
+    slideIndex++;
```

Comment:
Using postfix operator `++` is generally frowned upon in JavaScript. I would instead use `slideIndex += 1`